pandas/tests/io/excel/test_readers.py

```diff
@@ -1711,6 +1717,10 @@ def test_corrupt_files_closed(self, engine, read_ext):
             import xlrd
 
             errors = (BadZipFile, xlrd.biffh.XLRDError)
+        elif engine == "calamine":
+            from python_calamine import CalamineError
```

Comment:
I think more of a design decision for Calamine but are you expecting to implement more detailed error types? If so would probably be better to use those instead of a generic error like this

Reply:
I agree, I should split CalamineError in few detailed errors, I made a issue for future. Is it a blocker?

Reply:
No this is not a blocker